tests/integration/pom.xml

```diff
@@ -78,6 +78,20 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>mysql</artifactId>
+      <version>1.8.3</version>
```

Comment:
define `org.testcontainers:mysql` in root pom. so you don't need include version here.